pkg/restmapper/controllerrestmapper.go

```diff
@@ -0,0 +1,80 @@
+package restmapper
+
+import (
+	"fmt"
+
+	"k8s.io/apimachinery/pkg/api/meta"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/client-go/discovery"
+	"k8s.io/client-go/rest"
+)
+
+// NewControllerRESTMapper is the constructor for a ControllerRESTMapper
+func NewControllerRESTMapper(cfg *rest.Config) (meta.RESTMapper, error) {
```

Comment:
I'm assuming if you want to reset the cache you just reinstantiate a new controller restmapper?

Reply:
Yes, or it's a future enhancement!  For example, if we create a CRD, we might want to invalidate just the GV we created.